build_and_test_msvc.bat

```diff
@@ -0,0 +1,31 @@
+@echo off
+
+REM can be "debug" or "release"
+set config=%1
```

Comment:
Would it be possible to early out here if a config isn't passed in? Otherwise some calls below are going to shift a parameter "down". (For example, %source_files% will act as the output parameter for build_msvc.bat).

Reply:
Fixed.